### PR TITLE
Window controller tweaks

### PR DESCRIPTION
**Pull request by sindresorhus**
Just some minor simplifications.

### CODE REVIEW

File: Gifski/util.swift [hunk 1]
```diff
@@ -64,6 +64,55 @@ class SSView: NSView {
...
+		let size = defaultContentSize
+
+		guard let screen = NSScreen.main else {
+			return CGRect(origin: .zero, size: size)
+		}
+
+		let extraOffset = screen.visibleFrame.height * 0.19
+
+		return CGRect(
+			x: screen.visibleFrame.midX - (size.width / 2),
```
Comment (sindresorhus) [inline Gifski/util.swift]: @LarsJK Can you see anything wrong with my calculations here? When I remove `window.center()` below the window is not in the exact same `x` position...

File: Gifski/util.swift [hunk 2]
```diff
@@ -64,6 +64,55 @@ class SSView: NSView {
 }
 
 
+extension NSWindow {
+	static let defaultContentSize = CGSize(width: 480, height: 300)
+
+	static var defaultContentRect: CGRect = {
```
Comment (sindresorhus) [inline Gifski/util.swift]: @LarsJK Do you have any idea of how we can programmatically replicate the NIB window behavior of automatically stacking new windows not completely on top of each other?

screen shot 2018-03-19 at 12 57 15

File: Gifski/util.swift (hunk 1 above)
Comment (LarsJK) [reply Gifski/util.swift]: The calculations look correct to me.. Do you have the dock at the bottom or to either side?

File: Gifski/util.swift (hunk 2 above)
Comment (LarsJK) [reply Gifski/util.swift]: Not really.. I guess you would have to loop through `NSApplication.shared.windows`, or maybe you just need the keyWindow?
What is the standard cascading behaviour? Will it just add a new window at the same position as the keyWindow + the titles height?

File: Gifski/util.swift (hunk 1 above)
Comment (sindresorhus) [reply Gifski/util.swift]: No, the Dock is hidden.

Comment (sindresorhus) [reply Gifski/util.swift]: Ok, I solved it. I was stupid. I calculated based on the default window size, not our window size

File: Gifski/util.swift (hunk 2 above)
Comment (sindresorhus) [reply Gifski/util.swift]: I don't really know. I've added a TODO in the code. I'll look into it at some later time.

**Pull request merged**